packages/create-cloudflare/readme.md

```diff
@@ -0,0 +1,70 @@
+# create-cloudflare
+
+> Create new Cloudflare projects with one command
+
+> **Note**
+> This package has now been sunsetted! We recommend using the `wrangler generate [name] [template]` command to create new projects with templates.
+
+## Usage
+
+You may use `yarn`, `pnpm`, or `npm` to invoke the [`create-cloudflare`](https://www.npmjs.com/package/create-cloudflare) package:
+
+```
+$ npm init cloudflare <directory> <source> -- [options]
+# or
+$ pnpm create cloudflare <directory> <source> [options]
+# or
+$ yarn create cloudflare <directory> <source> [options]
+```
+
+> **Note:** All recent versions of `npm`, `yarn`, and `pnpm` support this feature!
+
+### Sources
+
+You may select the name of any subdirectory within the [`cloudflare/templates`](https://github.com/cloudflare/templates) repository to create your project; for example, `worker-typescript` and `examples/fast-google-fonts` are both valid subdirectory names.
```

Comment:
The URL to cloudflare/templates needs updating

Reply:
Package removed.